packages/components/src/shortcut/index.js

```diff
@@ -13,9 +24,7 @@ function Shortcut( { shortcut, className } ) {
 
 	if ( isString( shortcut ) ) {
 		displayText = shortcut;
-	}
-
-	if ( isObject( shortcut ) ) {
```

Comment:
This change concerns me a bit… we only expect to receive a string or object, but this protected against other values before and doesn't after this change.

How do you feel about reverting it?

Reply:
I will becuase I cannot reproduce the reason I made this change to begin with 🤦‍♀️ There was some issue with refining the type down to an object with the two properties, but it's working now, go figure.